src/encoded/types/file.py

```diff
@@ -979,6 +1000,39 @@ def _get_analysis_summary_fields(
             ),
         }
         return {key: value for key, value in to_include.items() if value}
+    
+    def _get_release_tracker_description(
+            self,
+            request_handler: RequestHandler,
+            file_properties: Dict[str, Any],
+        ) -> Union[str, None]:
+        """Get release tracker description for display on the home page."""
+        assay_title= get_unique_values(
+            request_handler.get_items(file_utils.get_assays(file_properties, request_handler)),
+            item_utils.get_display_title,
+            )
+        sequencer_title = get_unique_values(
+            request_handler.get_items(
+            file_utils.get_sequencers(file_properties, request_handler)),
+            item_utils.get_display_title,
+            )
+        file_format_title = get_property_value_from_identifier(
+                request_handler,
+                file_utils.get_file_format(file_properties),
+                item_utils.get_display_title,
+            )
+        if len(assay_title) > 1 or len(sequencer_title) > 1:
+            # More than one unique assay or sequencer
```

Comment:
Not sure it's a real case, but *could*, if multiple assays and/or sequencers, then if the same number of both, then just return the first one (?) - not sure that makes sense - just thinking ...

Reply:
@dmichaels-harvard I think the only instance where we'd expect there to be multiple assays or sequencers is for files like DSA or Truth Set files where multiple datasets were used to generate the file. For those special cases, do you think it is work including an override property to allow us to manually set this value?